speechbrain/lm/arpa.py

```diff
@@ -106,7 +97,6 @@ def read_arpa(fstream):
             `<logp> a quick red -23.4`
         And to access that here, use:
             `backoffs_by_order[3][('a', 'quick', 'red')]`
-
```

Comment:
Might I ask when you are changing these blank lines?

Reply:
I bet this is because he did not ran the pre-commit (which is fine, we can run them latter)

Reply:
I'll revert the change. I ran some linter accidentally.